### DISCUSSION TITLE
Multiple Hue Bridges & Error on Requesting Data

### DISCUSSION THREAD

**Discussion by Lorccan** [andesse/hue-clip-api.node-red-flows#42] [Q&A] [answered]
I have only just found your impressive set of flows. I think I understand what it's doing, but I wanted to confirm a couple of things:

 1. I have multiple hue bridges. I can see that the context vars are limited to the flow, so I assume I would just have a separate flow (with links in and out) for each bridge. Is that right?
 
2. When I inject Request all Data, I get errors from the "Motion Name, ID, Device Name" and "Button Name, ID, Device Name" nodes: "Please Update Devices First".  I'm not very conversant with the Hue CLIP API V2, so I am not clear how motion sensor devices and button devices differ from other devices such as lights.  Anyway, should I be concerned about these errors where I have a bridge that has neither a motion sensor nor a button?

**Comment by andesse** [comment, marked as answer]
Hi @Lorccan 

1. yes you figured this out. It works exactly like this. Add another SSE Client node for your other bridge. And connect it also to the discovery flows, where you request data, for the ids.

2. You can ignore any error messages when you request for devices. As an example it can occur when the request will search motion sensors but there aren't any connected.

Pretty sure you'll figure everything out. If you have further questions don't hesitate asking.

Andreas

**Reply by Lorccan** [reply]
Thanks for the super-quick response!
